cmd/genautocomplete/genautocomplete_bash.go

```diff
@@ -29,12 +30,23 @@ them directly
 
 If you supply a command line argument the script will be written
 there.
+
+If output_file is "-", then the output will be written to stdout.
 `,
 	Run: func(command *cobra.Command, args []string) {
 		cmd.CheckArgs(0, 1, command, args)
 		out := "/etc/bash_completion.d/rclone"
 		if len(args) > 0 {
-			out = args[0]
+			if args[0] == "-" {
+				err := cmd.Root.GenBashCompletion(os.Stdout)
+				if err != nil {
+					log.Fatal(err)
+				} else {
```

Comment:
Can you lose the `} else {` here? It is much easier to see the code always returns without it.

So

```go
if err != nil {
	log.Fatal(err)
}
return
```

Same in the other 2 places.

Reply:
Sure.